web/src/containers/BudgetSummary.js

```diff
@@ -59,9 +58,9 @@ class DataRow extends Component {
             const val = data[yr];
             return (
               <Fragment key={yr}>
-                <td className="mono right-align">{formatMoney(val)}</td>
-                <td className="mono right-align">{formatMoney(val * 0.9)}</td>
-                <td className="mono right-align">{formatMoney(val * 0.1)}</td>
+                <td className="mono right-align">{formatMoney(val.total)}</td>
+                <td className="mono right-align">{formatMoney(val.federal)}</td>
+                <td className="mono right-align">{formatMoney(val.state)}</td>
```

Comment:
Doing these calculations in the budget reducer instead

Reply:
💥